examples/jupyter/demo010_advection.ipynb

```diff
@@ -0,0 +1,218 @@
+      "    @ ~/Projects/LFAToolkit.jl/examples/jupyter/demo010_advection.ipynb:15\n",
+      "  [2] eval\n",
+      "    @ ./boot.jl:373 [inlined]\n",
+      "  [3] include_string(mapexpr::typeof(REPL.softscope), mod::Module, code::String, filename::String)\n",
+      "    @ Base ./loading.jl:1196\n",
+      "  [4] #invokelatest#2\n",
+      "    @ ./essentials.jl:716 [inlined]\n",
+      "  [5] invokelatest\n",
+      "    @ ./essentials.jl:714 [inlined]\n",
+      "  [6] (::VSCodeServer.var\"#164#165\"{VSCodeServer.NotebookRunCellArguments, String})()\n",
+      "    @ VSCodeServer ~/.vscode/extensions/julialang.language-julia-1.6.24/scripts/packages/VSCodeServer/src/serve_notebook.jl:19\n",
+      "  [7] withpath(f::VSCodeServer.var\"#164#165\"{VSCodeServer.NotebookRunCellArguments, String}, path::String)\n",
+      "    @ VSCodeServer ~/.vscode/extensions/julialang.language-julia-1.6.24/scripts/packages/VSCodeServer/src/repl.jl:184\n",
+      "  [8] notebook_runcell_request(conn::VSCodeServer.JSONRPC.JSONRPCEndpoint{Base.PipeEndpoint, Base.PipeEndpoint}, params::VSCodeServer.NotebookRunCellArguments)\n",
+      "    @ VSCodeServer ~/.vscode/extensions/julialang.language-julia-1.6.24/scripts/packages/VSCodeServer/src/serve_notebook.jl:13\n",
+      "  [9] dispatch_msg(x::VSCodeServer.JSONRPC.JSONRPCEndpoint{Base.PipeEndpoint, Base.PipeEndpoint}, dispatcher::VSCodeServer.JSONRPC.MsgDispatcher, msg::Dict{String, Any})\n",
+      "    @ VSCodeServer.JSONRPC ~/.vscode/extensions/julialang.language-julia-1.6.24/scripts/packages/JSONRPC/src/typed.jl:67\n",
+      " [10] serve_notebook(pipename::String, outputchannel_logger::Base.CoreLogging.SimpleLogger; crashreporting_pipename::String)\n",
+      "    @ VSCodeServer ~/.vscode/extensions/julialang.language-julia-1.6.24/scripts/packages/VSCodeServer/src/serve_notebook.jl:136\n",
+      " [11] top-level scope\n",
+      "    @ ~/.vscode/extensions/julialang.language-julia-1.6.24/scripts/notebook/notebook.jl:32\n",
+      " [12] include(mod::Module, _path::String)\n",
+      "    @ Base ./Base.jl:418\n",
+      " [13] exec_options(opts::Base.JLOptions)\n",
+      "    @ Base ./client.jl:292\n",
+      " [14] _start()\n",
+      "    @ Base ./client.jl:495"
+     ]
+    }
+   ],
+   "source": [
+    "# setup\n",
+    "p = 4\n",
+    "dimension = 1\n",
+=======
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# setup\n",
+    "p = 1\n",
+    "dimension = 2\n",
+>>>>>>> 48dc89b931d819c8a441fe77fbbf998b0a57a77b
+    "mapping = hale_trefethen_strip_transformation(1.4)\n",
+    "\n",
+    "mesh = []\n",
+    "if dimension == 1\n",
+    "   mesh = Mesh1D(1.0)\n",
+    "elseif dimension == 2\n",
+    "   mesh = Mesh2D(1.0, 1.0)\n",
+    "end\n",
+    "\n",
+    "# operator\n",
+    "mass = GalleryOperator(\"mass\", p + 1, p + 1, mesh, mapping = mapping)\n",
+    "advection = GalleryOperator(\"advection\", p + 1, p + 1, mesh, mapping = mapping)"
+   ]
+  },
```

Comment:
1) merge conflicts

2) output should be cleared before committing the  jupyter notebooks